easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.47.1-foss-2016a.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'GObject-Introspection'
+version = '1.47.1'
+
+homepage = 'https://wiki.gnome.org/GObjectIntrospection/'
+description = """GObject introspection is a middleware layer between C libraries
+ (using GObject) and language bindings. The C library can be scanned at
+ compile time and generate a metadata file, in addition to the actual
+ native C library. Then at runtime, language bindings can read this
+ metadata and automatically provide bindings to call into the C library."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+
+pyver = '2.7.11'
```

Comment:
@hajgato no need for this, just include the Python version directly in `dependencies`?